packages/multy.rb

```diff
@@ -3,16 +3,16 @@
 class Multy < Package
   description 'Easily deploy multi cloud infrastructure.'
   homepage 'https://multy.dev/'
-  version '0.1.57'
+  version '1.0.1'
```

Comment:
This is not the correct version.  See https://github.com/multycloud/multy/releases.